templates/esm/index.html

```diff
@@ -14,8 +14,8 @@
       <span class="u-sv3"><h1>Extended Security Maintenance</h1></span>
       <span class="u-sv3"><p class="p-heading--four">Security updates for 14.04 LTS until 2022</p></span>
       <p>Continue to receive security updates for the Ubuntu base OS and critical infrastructure components &ndash; Ceph, OpenStack and more &ndash; with ESM in your Ubuntu Advantage subscription.</p>
-      <p>Ubuntu Advantage for Infrastructure is available for physical servers, virtual machines, containers, and desktops.</p>
-      <p><a href="https://buy.ubuntu.com/" class="p-button--positive"><span class="p-link--external">Get started with Ubuntu Advantage</span></a></p>
+      <p>Ubuntu Advantage for Infrastructure is available for physical servers, virtual machines, containers, and desktops. Free for personal use.</p>
+      <p><a href="/advantage#esm" class="p-button--positive">Get started with Ubuntu Advantage</a></p>
```

Comment:
```suggestion
      <p><a href="/advantage" class="p-button--positive">Get started with Ubuntu Advantage</a></p>
```



I really do love the highlighting of the relevant row in the table.

Unfortunately, because the table is specifically for paid subscriptions, it makes sense only for a link that’s specifically about paid subscriptions. And changing “And it’s free for personal use” to “Free for personal use” does not, in any way, make this section specifically about personal subscriptions. So with the page as currently structured, I don’t think we can use the row highlighting here.

(Separately I’ve reported #6110 on the behaviour of the anchors.)